src/pybkgmodel/processing.py

```diff
@@ -908,3 +909,40 @@ def bkg_map_maker(self, maker):
         if not isinstance(maker, ExclusionMap):
             raise TypeError(f"Maker must be of type {ExclusionMap}")
         BkgMakerBase.bkg_map_maker.fset(maker)
+
+class RunwiseOffDataMap(_Runwise):
+    def __init__(self, off_runs, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.off_runs = off_runs
+        self.excl_region    = [Regions.parse(reg,format='ds9') for reg in 
+                               excl_region]
+        self._bkg_map_maker = OffDataMap(off_runs=self.off_runs,
+                                        x_edges=self.x_edges,
+                                        y_edges=self.y_edges,
+                                        e_edges=self.e_edges,
+                                        regions=self.excl_region,
+                                        cuts=self.cuts,
+                                        pointing_delta=self.pointing_delta
+                                        )
+
+class StackedOffDataMap(_Stacked):
+    def __init__(self, *args, off_files, **kwargs):
+        super().__init__(*args, **kwargs)       
+        self.off_files      = glob.glob(off_files)
+        self.off_runs       = tuple(
+                                filter(
+                                    lambda r: r.obs_id is not None, 
+                                    [RunSummary(fname) for fname in 
+                                    self.off_files]
+                                    )
+                                )
+        self.excl_region    = [Regions.parse(reg,format='ds9') for reg in 
```

Comment:
Suggest removing the line break after `in`.

Reply:
Same as above, I can break the line at a different place. Too long line disagrees with coding guidelines. No strong personally opinion on it.